Tools/Pipeline/Controls/PropertyGridTable.cs

```diff
@@ -196,7 +196,7 @@ private void Drawable_Paint(object sender, PaintEventArgs e)
             if(_edit)
             {
                 if (!Global.Unix)
-                    _selectedCell.Edit(pixel1);
+                    _selectedCell?.Edit(pixel1);
```

Comment:
@cra0zy a screenshot of it happening: http://imgur.com/HQNNATo

This happens every time I open a `mgcb` file, then click on the root node, scroll down to references,  and double click `collection`.